### PR TITLE
packaging

### PR DESCRIPTION
**Pull request by mathematicalmichael**
First off, awesome project. Web scraping has always intimidated me but I was able to get this up and running in no time and thought I'd make a contribution of my own since I've been in the midst of packaging up a lot of thesis-related work so the process is fresh in my mind.

`pip install -e .` for a local development installation
`pip install -e .[testing]` to get the testing dependencies

`pytest` work now from root directory without hard-coding path to the chrome binary.
I added instructions to `README` for a linux install of chrome. I couldn't get this running in a containerized environment because of the need to launch chrome, but it worked great in a new virtual environment.

If this looks good to you, I can help get it up on `PyPi`. This PR sets up everything for the packaging structure they expect. I could also set up a Github Actions workflow that lets you publish versions after the initial one right from Github using Releases. 

please feel free to request any changes to the `README`, I kind of cobbled that together at the end.

I guessed at your email and desired licensing. 

I'd love to add CI to this but I would need to figure out how to run headless chrome to do that since Github Actions run in containerized environments. I'm sure it's possible but I'm not familiar with the process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,22 @@ for legislation related to human rights issues.
...
 open comparative datasets on legislation around the world. 
 
+
+## Installation
+`legiscrapor` can be installed with
+
+```sh
+pip install .
+```
+(PyPi installation available soon)
```
Comment (mathematicalmichael) [inline README.md:16]: perhaps presumptuous but if you put all this work in, only a little bit more to get `pip install legiscrapor` + globally available command line functionality like (e.g. `legiscrape ...` or whatever, c.f. line 61 in `setup.cfg`)

**Comment by mathematicalmichael** [comment]
Oh I guess I need to update the usage patterns + test those out. But I have some questions about that (like, can I remove the chromedriver path as a necessary argument? make it optional?)

That's going to require global import statements rather than local ones (update: fixed). But the idea is that instead of `python src/legiscrapor/run_generic.py ...` we could just have the script available so the syntax is something like `scrape_generic ...` (doesn't have to match the filename).

**Comment by mathematicalmichael** [comment]
last commits: missed a lot of dependencies. now able to run all the `run_` scripts without import errors =) `pytest` passing was a false sense of security lol, but now I think this is close to buttoned-up.

**Comment by amrosnik** [comment]
@mathematicalmichael wowowow, I continue to be pleased that you want to contribute to this project already! I just pushed a `dev` branch -- could you change this PR, or close it and open a new one so we can merge into `dev` before `main` ?

File: README.md (hunk 1 above)
Comment (amrosnik) [reply README.md:16]: oh that's a cool idea...I definitely want to think about this more, but at the very least get this initial packaging tried out first!

**Pull request merged**